Controller/articles.controller.js

```diff
@@ -25,3 +27,22 @@ exports.getAllArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVoteById = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const updateBody = req.body;
+
+  const promises = [updateArticleVoteById(articleId, updateBody)];
+
+  if (articleId) {
```

Comment:
This will always exist. You have defined an endpoint with a parametric id that represents anything put in that location. Valid or not, the variable articleId will always have a value and this if condition will never be falsy.